frontend/src/components/maps/MineMap.js

```diff
@@ -83,7 +84,75 @@ class MineMap extends Component {
       "esri/widgets/Expand",
       "esri/widgets/ScaleBar",
       "esri/widgets/Legend",
-    ]).then(([LayerListWidget, Expand, ScaleBar, Legend]) => {
+      "esri/layers/GroupLayer",
+    ]).then(([LayerListWidget, Expand, ScaleBar, Legend, GroupLayer]) => {
+      const administrativeLayer = new GroupLayer({
+        title: "Administrative Boundaries",
+        visible: true,
+      });
+      const tenureLayer = new GroupLayer({
+        title: "Mineral, Placer, and Coal",
+        visible: true,
+      });
+      const roadsLayer = new GroupLayer({
+        title: "Roads",
+        visible: true,
+      });
+
+      const adminLayerArray = [
+        "Indian Reserves & Band Names",
+        "BC Mine Regions",
+        "Natural Resource Regions",
+        "Natural Resource Regions - WMS",
+        "Land Status and Survey Parcels - PMBC",
+        "Regional Districts - WMS",
+        "Ministry of Transportation District Boundaries - WMS",
+        "Municipality Boundaries",
+      ];
+      const tenureLayerArray = [
+        "Coal Licence Applications",
+        "Coal Leases",
+        "Coal Licences",
+        "Mining Leases",
+        "Mineral Claims",
+        "Placer Leases",
+        "Placer Claims",
+      ];
+      const roadLayerArray = ["Roads DRA", "Forest Tenure Roads"];
+
+      adminLayerArray.forEach((layerTitle) => {
+        const newTenureLayer = view.map.allLayers.find(({ title }) => title === layerTitle);
+        if (newTenureLayer) {
+          administrativeLayer.add(newTenureLayer);
+        }
+      });
+      tenureLayerArray.forEach((layerTitle) => {
+        const newTenureLayer = view.map.allLayers.find(({ title }) => title === layerTitle);
+        if (newTenureLayer) {
+          tenureLayer.add(newTenureLayer);
+        }
+      });
+      roadLayerArray.forEach((layerTitle) => {
+        const newTenureLayer = view.map.allLayers.find(({ title }) => title === layerTitle);
+        if (newTenureLayer) {
+          roadsLayer.add(newTenureLayer);
+        }
+      });
```

Comment:
This is looking a lot better than the WIP code we had yesterday!

Since these all do the exact same thing, how about creating a function to handle it? Something like:
```javascript
const addLayers = (layerArray) =>
    layerArray.forEach((layerTitle) => {
        const newTenureLayer = view.map.allLayers.find(({ title }) => title === layerTitle);
        if (newTenureLayer) {
          roadsLayer.add(newTenureLayer);
        }
      });

addLayers(adminLayerArray);
addLayers(tenureLayerArray);
addLayers(roadLayerArray);
```

Reply:
done